python/desc/sims/GCRCatSimInterface/DatabaseEmulator.py

```diff
@@ -167,6 +223,8 @@ def _init_data_indices(self):
         if self._chunk_size is None:
             self._chunk_size = self._data_indices.size
 
+        return native_filters
```

Comment:
both `self._data_indices` and `self._chunk_size` now depend on `native_filters`, so all should be returned or stored in `self`.